src/cli.js

```diff
@@ -13,6 +13,10 @@ function getArgs() {
     )
     .describe('className', 'Create a React component (wraps JSX in React.createClass call)')
     .alias('className', 'c')
+    .boolean('e')
+    .describe('e', 'Decide whether to wrap in module.exports')
```

Comment:
Cheers, I resubmitted the braces change in a separate PR.  The other changes are submitted as before.  I was not sure how to break the second set of changes into it's own PR without dependence on the earlier commits, so I left it alone.